pd/src/genesis.rs

```diff
@@ -116,7 +116,7 @@ impl TryFrom<pb::genesis_app_state::ValidatorPower> for ValidatorPower {
 impl Protobuf<pb::genesis_app_state::ValidatorPower> for ValidatorPower {}
 
 /// The application state at genesis.
-#[derive(Deserialize, Serialize, Debug, Clone)]
+#[derive(Deserialize, Serialize, Debug, Clone, Default)]
```

Comment:
The genesis app state can derive `Default` rather than implementing it directly.